plane/source/docs/advanced-failsafe-configuration.rst

```diff
@@ -262,6 +262,13 @@ the aircraft is a danger to people or other aircraft. To force an
 immediate termination you should use the :ref:`AFS_TERMINATE<AFS_TERMINATE>` parameter. By
 setting that parameter to 1 the aircraft will immediately terminate.
 
+AFS_OPTIONS
+------------------
+In addition, the behavior of AFS can be modified by the setting of the :ref:`AFS_OPTIONS<AFS_OPTIONS>` bitmask parameter (no bits are set by default):
```

Comment:
```suggestion
In addition, the behavior of AFS during GCS communication loass can be modified by the setting of the :ref:`AFS_OPTIONS<AFS_OPTIONS>` bitmask parameter (no bits are set by default):
```
based on your change this param only affects GCS loss....not GPS(which also changes mission sequence)

Reply:
I'm not sure if we should change this sentence though. In the future, someone could use other bit mask to change behavior of RC failsafe or GCS failsafe.

Reply:
if and when the behavior is changed, we can modify the sentence...as it is it could be assumed to be used for GPS mission changes also, which It is not